infrastructure/lambda/template.yaml

```diff
@@ -111,12 +111,31 @@ Globals:
       - UsePermissionsBoundary
       - !Ref PermissionsBoundary
       - !Ref AWS::NoValue
+    # A minimum 1.5GB is recommended for Java
     Timeout: 40
     MemorySize: !FindInMap [ MemorySizeMapping, Environment, !Ref 'Environment' ]
+    Runtime: java17
```

Comment:
We are still only on java 11 in passport

Reply:
Sorted. Back to java11